### PR TITLE
Initial Batch 7 (1)

### PR DESCRIPTION
**Pull request by ArthurMa1978**

### CODE REVIEW

File: sdk/advisor/Azure.ResourceManager.Advisor/ci.mgmt.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```
Comment (Yao725) [inline sdk/advisor/Azure.ResourceManager.Advisor/ci.mgmt.yml:1]: Why do we have two ci.mgmt.yml files in this RP?

**Review by Yao725 — COMMENTED** [review]
I think we also need to run this [command](https://github.com/Azure/azure-sdk-for-net/blame/main/eng/templates/README.md#L30) to update the yml in resource manager.

File: sdk/advisor/Azure.ResourceManager.Advisor/ci.mgmt.yml (hunk 1 above)
Comment (ArthurMa1978) [reply sdk/advisor/Azure.ResourceManager.Advisor/ci.mgmt.yml]: fixed

**Review by Yao725 — APPROVED** [review]

**Pull request merged**